src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/OpenApiSchemaExtensions.cs

```diff
@@ -229,40 +252,38 @@ private static void ApplyBase64Attribute(OpenApiSchema schema)
 
     private static void ApplyRangeAttribute(OpenApiSchema schema, RangeAttribute rangeAttribute)
     {
-#if NET
+        schema.Maximum = decimal.TryParse(rangeAttribute.Maximum.ToString(), out decimal maximum)
```

Comment:
Why parse with the current culture, then format using invariant? I think it should take [RangeAttribute.ParseLimitsInInvariantCulture](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.rangeattribute.parselimitsininvariantculture?view=net-9.0) and [RangeAttribute.ConvertValueInInvariant](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.rangeattribute.convertvalueininvariantculture?view=net-9.0) into account.

Reply:
I tried to follow the current behaviour, rather than do what I would do in a vacuum with no weight of backwards compatibility (which would be to be invariant without exception).

Reply:
The current behavior is that these types are numeric, not strings, so there's nothing to follow. Always invariant is simply wrong; the properties on RangeAttribute exist for a reason.